### PR TITLE
SDIT-1215 Remove SYSTEM_USER from methods where not used

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java [hunk 1]
```diff
@@ -181,7 +181,7 @@ public int getEnrouteOffenderCount(final String agencyId, final LocalDate date)
         return movementsRepository.getEnrouteMovementsOffenderCount(agencyId, defaultedDate);
     }
 
-    @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER"})
```
Comment (karenmillermoj) [inline src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java:184]: These 2 methods are only called by prison-staff-hub which does not have SYSTEM_USER role

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+import org.springframework.web.util.UriComponentsBuilder
+import uk.gov.justice.hmpps.prison.executablespecification.steps.AuthTokenHelper.AuthToken
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalDateTime.now
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+import java.util.Map
+
+class MovementResourceTest : ResourceTest() {
```
Comment (karenmillermoj) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt:23]: converted to kotlin and only added authorisation tests - additional tests exist as cucumber tests, these check the actual values returned

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+      GET,
+      createHttpEntity(token, null),
+      object : ParameterizedTypeReference<String>() {},
+      "LEI",
+    )
+    assertThatStatus(response, 200)
+    assertThatJson(response.body).isEqualTo("2")
+  }
+
+  @Nested
```
Comment (karenmillermoj) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt:161]: converted to kotlin and only added authorisation tests for  `/api/movements/{agencyId}/in/{isoDate}` and  `/api/movements/rollcount/{agencyId}/in-reception` - additional tests exist as cucumber tests, these check the actual values returned

File: src/main/java/uk/gov/justice/hmpps/prison/repository/MovementsRepository.java [hunk 4]
```diff
@@ -70,20 +70,6 @@ public List<Movement> getRecentMovementsByDate(final LocalDateTime fromDateTime,
         return movements.stream().map(MovementDto::toMovement).collect(Collectors.toList());
     }
 
-
```
Comment (karenmillermoj) [inline src/main/java/uk/gov/justice/hmpps/prison/repository/MovementsRepository.java:73]: Removal - as not required (accidentally missed from previous PR)

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**